build/release.js

```diff
@@ -27,6 +29,15 @@ module.exports = function( Release ) {
 			ensureSizzle( Release, callback );
 		},
 		/**
+		 * Set the version in the src folder for distributing AMD
+		 */
+		_setSrcVersion: function() {
+			var corePath = __dirname + "/../src/core.js",
+				contents = fs.readFileSync( corePath, "utf8" );
+			contents = contents.replace( /@VERSION/g, Release.newVersion );
```

Comment:
:smile_cat: True but it's a question of complexity. I know how a regex works very well, but if you pull in a template package I will have to look up how it works. At some point I agree it might make sense to put one in, but to me it doesn't look like we are at that point yet. The code is all very understandable and we haven't had problems with it.